doc/sphinx-guides/source/user/dataverse-management.rst

```diff
@@ -69,7 +69,8 @@ The Theme features provides you with a way to customize the look of your Dataver
 
 - Inherit the theme from the parent Dataverse collection. This option is helpful if you'd like consistency across several Dataverse collections that all share the same parent.
 - Add or update a logo image, which will appear at the top of your Dataverse collection.
-- Add or update a footer image, which will appear at at the bottom of your Dataverse collection.
+- Add or update a thumbnail image, which will appear on featured dataverses of your Dataverse collection.
```

Comment:
Interesting feature! My first thought is, should this feature be available via API? That way the new frontend can use it some day.

Also, these issue are related:

- #10194
- #10242

Reply:
In order to have thumbnail logo via API, all Theme feature have to be moved before. Do you know if the development of #10194 is well advanced and PR will be open soon ?

Either I wait for PR #10194 to be accepted (and I adapt the source code from it), or I propose one. But either way, I should change the status of this PR (#10433) to draft.

Reply:
I got the impression from @DS-INRA that he might have a team member like you work on #10194 but maybe I'm wrong! 😅